client/cos.go

```diff
@@ -259,6 +259,8 @@ func (c *CosignerProvider) ValidateCos(cosSig []byte, expectedNonce string, expe
 	return nil
 }
 
+// CreateInitAuthSig generates a random nonce, validates the redirectURI, creates an InitMFAAuth message,
+// marshals it to JSON, and returns the JSON message along with the nonce.
```

Comment:
I really appreciate you adding comments to the GoDocs. I am a fan of [Semantic Linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/)  which is what is looks like your doing. 

> There is no rigid line length limit in Go code, but avoid uncomfortably long lines. Similarly, don’t add line breaks to keep lines short when they are more readable long–for example, if they are repetitive.

https://go.dev/wiki/CodeReviewComments#line-length

Try to avoid comments (and code) lines that extend beyond column 80, unless doing so would not be awkward. Lines 80 columns or less are nicer to read in an editor or diff without the editor doing word wrapping.

```suggestion
// CreateInitAuthSig generates a random nonce, validates the redirectURI, 
// creates an InitMFAAuth message, marshals it to JSON, 
// and returns the JSON message along with the nonce.
```